lib/chef/resource/locale.rb

```diff
@@ -24,6 +24,7 @@ class Locale < Chef::Resource
 
       description "Use the locale resource to set the system's locale."
       introduced "14.5"
+      provides :locale, platform_family: %w{rhel fedora amazon debian}
```

Comment:
we've been going away from doing this.  stuff like bff_package being still declared on linux is useful for chefspec for example.